SolutionInfo.cs

```diff
@@ -43,5 +43,5 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("9.2.0.0")]
```

Comment:
Please avoid committing the solutions file into the changes. This might break whole website.